_includes/head.html

```diff
@@ -12,9 +12,7 @@
     <link rel="shortcut icon" href="{{site.url}}{{site.baseurl}}/favicon.ico" type="image/x-icon">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Is this still needed if we're dropping material design?

Reply:
i dropped them also. looks good